Database/initDB/schema.sql

```diff
@@ -0,0 +1,16 @@
+-- Create the Song table
+CREATE TABLE Song (
+                      id SERIAL PRIMARY KEY,
+                      name VARCHAR(255) NOT NULL,
```

Comment:
Are 255 charcters enough for urls? soundcloud links can get very long in some cases. is this easily extenable to say 520 or can this produce problems?

Reply:
would leave it at 255 first, if really needed it can be changed later, but 255 is default for databases